pages/favorite.js

```diff
@@ -1,12 +1,36 @@
 import Header from "../components/Header/Header";
 import Footer from "../components/Footer/Footer";
+import styled from "styled-components";
+import { Fragment } from "react";
+import FoodCard from "../components/Food/FoodCard";
 
-export default function Favorite() {
+export default function Favorite({ foods, onToggleFavorite }) {
+  const favoriteFoods = foods.filter((food) => food.isFavorite === true);
+  const favoriteFoodsList = favoriteFoods.map((favoriteFood) => (
+    <Fragment key={favoriteFood.id}>
+      <FoodCard
+        id={favoriteFood.id}
+        name={favoriteFood.name}
+        onToggleFavorite={onToggleFavorite}
+        foods={favoriteFood}
+      />
+    </Fragment>
+  ));
   return (
     <>
       <Header />
-      <h1>Favoriten</h1>
+      <StyledFavoriteHeader>Favoriten</StyledFavoriteHeader>
+
+      {favoriteFoodsList}
```

Comment:
I was a little confused why this isn't in an html element but it probably works because the Fragment and FoodCard in line 10 and 11 are what is included here, right?

Reply:
Exactly! {favoriteFoodsList} and its method is written in line 9-17. Therefore, It's just needed to pass the const in the return statement.